main.go

```diff
@@ -97,25 +72,8 @@ func main() {
 }
 
 func applyCacheConfig(cacheConfig string) error {
-	switch cacheConfig {
-	case cacheInputNone:
-		return applyNoCache()
-	case cacheInputAll:
+	if cacheConfig == cacheInputAll {
```

Comment:
this constant is used on only one place, and when you define a string without a variable that is a constant also, so in this case I would move the constant value here, near to the check and remove the constant

Reply:
Removed.